### PR TITLE
Updates to Weather.vue

### PR DESCRIPTION
**Pull request by BrandonSokolowskiWSU**
Added a few things to the page, and summarized in the commit messages.

### CODE REVIEW

File: src/views/Weather.vue [hunk 1]
```diff
@@ -1,30 +1,53 @@
...
             <!-- Weather Display -->
-            <p>Condition: {{ weather_main }}, {{ weather_description }}</p> <!-- Display weather description -->
-            <p>Temperature: {{ convert_kelvin_to_fahrenheit(result.temp) }}°F ({{ convert_kelvin_to_celsius(result.temp) }}°C)</p> <!-- Displays temp in F and C -->
-            <p>Feels Like: {{ convert_kelvin_to_fahrenheit(result.feels_like) }}°F ({{ convert_kelvin_to_celsius(result.feels_like) }}°C)</p> <!-- Displays feels like in F and C -->
-            <p>Humidity: {{ result.humidity }}%</p> <!-- Displays air humidity -->
-            <p>Pressure: {{ result.pressure }} hPa</p> <!-- Displays air pressure in hPa -->
-            <p>Wind Speed: {{ convert_mps_to_mph(wind_speed) }} mph ({{ wind_speed }} m/s)</p> <!-- Displays wind speed in mph and m/s -->
-            <p>Wind Direction: {{ wind_direction }}°</p> <!-- Display wind direction -->
-            <p>Cloudiness: {{ clouds_percent }}%</p> <!-- Display cloud coverage -->
+            <img v-if="weather_icon" :src="`https://openweathermap.org/img/wn/${weather_icon}@2x.png`" alt="Weather Icon" /> <!-- Display weather icon -->
```
Comment (mrago256) [inline src/views/Weather.vue:12]: This is a really cool addition!

**Review by mrago256 — APPROVED** [review]
These are some really good changes! One idea I had, I know that you mentioned that you would like to use the map view to get weather for a selected location, but another way you could expand this in the future is to show the weather data for locationes saved in the selected trip once we get them saved. Like have somewhere on the page that has each city name and then weather for them.

File: src/views/Weather.vue [hunk 2]
```diff
@@ -1,30 +1,53 @@
...
+            <p></p>
             <p>Weather provided by OpenWeather</p>
         </div>
         <div v-else>
+            <p></p>
             <p>No weather data found for {{ city }}</p> <!-- Error Handling -->
         </div>
     </div>
+    <!-- MapView component to display the map -->
+    <MapView ref="mapView" />
```
Comment (CCooley33) [inline src/views/Weather.vue:31]: We'll talk about storing the destination's coordinates in the store for use with this in todays meeting

**Review by CCooley33 — APPROVED** [review]
Good work looking forward to getting the map working for this

**Pull request merged**